core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandler.java

```diff
@@ -676,6 +650,14 @@ private void processErrorResponse(Error errorMessage) {
                 throttler,
                 sessionMetricUpdater,
                 logPrefix);
+        PrepareRequest reprepareRequest = Conversions.toPrepareRequest(reprepareMessage);
+        long reprepareStartNanos = System.nanoTime();
+        trackNodeEnd(
+            this,
+            defaultExecutionInfo().withServerResponse(errorFrame).build(),
+            new IllegalStateException("Unexpected response " + errorMessage));
+        // TODO: Shall we have different logPrefix?
```

Comment:
are you thinking this could be useful just to make it more clear that the query is being reprepared?